### ISSUE TITLE
Potential new sub-lineage of B.1.617.2 with Spike N501Y AA in Turkey

### ISSUE THREAD

**Issue by bikudh** [cov-lineages/pango-designation#185]
Sub-lineage of: B.1.617.2 with Spike_N501Y, mostly in Turkey

Earliest sequence: 28/03/2021 (Romania EPI_ISL_2100241)
Most recent sequence: 06/08/2021 (Turkey EPI_ISL_3405561)

Countries circulating: Mostly Turkey in the recent days but seen in sequences from other countries in the list:
 **Turkey**
 United Kingdom 
 Romania 
 Germany 
 India 
 USA 
 Spain 
 Bangladesh 
 South Africa 
 Poland 
 Czech Republic 
 Italy 
 Mexico 
 Japan 
 Pakistan 

According to data from GISAID, the earlier sequences of B.1.617.2+Spike_N501Y were detected in Romania as early as 03/28/2021. The strain was introduced in Turkey in late May (5/25/2021) and it took off in Turkey in the middle of June, and it is mostly seen in Turkey very recently. 

425 genomes in GISAID; Atleat 310 from Turkey.
[B.1.617.2 with Spike_N501Y Turkey.txt](https://github.com/cov-lineages/pango-designation/files/6989153/B.1.617.2.with.Spike_N501Y.Turkey.txt)

Only a handful of these sequences used to generate the tree
![image](https://user-images.githubusercontent.com/88990329/129497901-564f868f-b304-48b7-be3f-27673f94e3cb.png)

Proposed lineage name
AY.13 ( if not taken)

**Comment by zach-hensel** [comment]
The concentration of Delta + N501Y + E484K sequences in Turkey with few of them clustering to any one part of the UShER tree and some having additional abnormal Delta mutations makes this seem pretty prone to artifacts. For example, the most recent sample date with Delta+N501Y+E484K is 5/August and there are:
* 2 sequences with Nsp6 del106/108
* 6 sequences with no Nsp6 mutations
* 2 sequences with Nsp6 T77A

I haven't looked at the N501Y (but not E484K) sequences; maybe those are more reasonable.

**Comment by bikudh** [comment]
Agree! We will watch for dump of new sequences from Turkey or elsewhere to see how the prevalence is affected. These sequences account for about 7% of sequences they have deposited so far.

**Comment by bikudh** [comment]
A few new sequences are submitted in GISAID.

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/srlbkd/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/srlbkd/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



Virus name | Accession ID | Collection date | Submission Date
-- | -- | -- | --
hCoV-19/Greece/10207739/2021 | EPI_ISL_3545332 | 7/8/2021 | 8/21/2021
hCoV-19/Turkey/HSGM-C9077/2021 | EPI_ISL_3540401 | 8/2/2021 | 8/20/2021
hCoV-19/Turkey/HSGM-C8865/2021 | EPI_ISL_3540295 | 8/2/2021 | 8/20/2021
hCoV-19/Turkey/HSGM-C8898/2021 | EPI_ISL_3540284 | 8/2/2021 | 8/20/2021
hCoV-19/Turkey/HSGM-C8890/2021 | EPI_ISL_3540274 | 8/2/2021 | 8/20/2021
hCoV-19/Germany/BY-RKI-I-197647/2021 | EPI_ISL_3496702 | 7/20/2021 | 8/19/2021
hCoV-19/England/QEUH-1B1ADFF/2021 | EPI_ISL_3444779 | 8/5/2021 | 8/16/2021



</body>

</html>
